public/index.html

```diff
@@ -6,15 +6,15 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
     <link rel="stylesheet" type="text/css" href="css/style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script type="text/javascript" src="validation.js"></script>
+    <script type="text/javascript" src="js/validation.js"></script>
```

Comment:
@Sjones352 - does validation.js make sense as a name for this file as you've grown this program? if not, please rename the file to a more fitting name :).